src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -275,46 +275,46 @@ private void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepos
         }
         listener.getLogger().format("%n  %d branches were processed%n", branches);
 
-            listener.getLogger().format("%n  Getting remote pull requests...%n");
```

Comment:
If that is the _only_ think you are changing, better to do a separate PR (or simply direct commit) that fixes the indentation, so as not to confuse PR reviews.